lib/mongoid/changeable.rb

```diff
@@ -161,12 +161,24 @@ def attribute_change(attr)
     #   model.attribute_changed?("name")
     #
     # @param [ String ] attr The name of the attribute.
+    # @param **kwargs The optional keyword arguments.
+    #
+    # @option **kwargs [ Object ] :from The object the attribute was changed from.
+    # @option **kwargs [ Object ] :to The object the attribute was changed to.
```

Comment:
Did we agree on `**kwargs` here in the documentation conventions PR?

Reply:
I think we did. I checked back there when I decided on the naming.

Reply:
We didn't do this before and I think omitting `**kwargs` wouldn't add any ambiguity, if we don't need that extra word I'd rather not have it.

Reply:
It's been documented here: https://github.com/mongodb/mongoid/blob/49ff14f4a2187ef61abd81ecbc5a0fcd043f2401/docs/contributing/code-documentation.txt#L285